addons/website/static/src/scss/website.scss

```diff
@@ -1029,18 +1029,19 @@ header {
             // Hack: padding is used by layout option (boxed, etc) so use
             // border here to be able to combine the effect.
             @if $-hamburger-right {
-                border-right: o-website-value('sidebar-width') solid transparent;
+                padding-right: o-website-value('sidebar-width');
             } @else {
                 border-left: o-website-value('sidebar-width') solid transparent;
             }
 
             > header {
                 @if $-hamburger-right {
                     @include o-position-absolute(0, 0, 0, auto);
+                    position: absolute;
                 } @else {
                     @include o-position-absolute(0, auto, 0, 0);
+                    position: fixed;
```

Comment:
There is no reason for the menu to be CSS-positioned differently depending on if it is on the left or the right. On top of that, you literally wrote `position: absolute` followed by `position: absolute`...

Keep everything `fixed`, especially in a stable version.